test/TestGrainInterfaces/IFilteredImplicitSubscriptionGrain.cs

```diff
@@ -1,6 +1,4 @@
 ﻿using Orleans;
-using Orleans.Streams;
```

Comment:
I think you may need to merge, this looks out of date.
Predicates need to be in interfaces to be available in client.